include/ck/tensor_operation/gpu/device/impl/device_batched_gemm_multiple_d_dl.hpp

```diff
@@ -649,6 +645,8 @@ struct DeviceBatchedGemmMultipleD_Dl : public DeviceBatchedGemmMultiD<ALayout,
 
     static bool IsSupportedArgument(const Argument& arg)
     {
+        assert(arg.K % K1 == 0);
```

Comment:
Moving `assert` statement is not an appropriate solution. You should strive to **not** use `assert` statements **at all**, anywhere in the code. The reason is as I said earlier - in `Release` build mode this check is not executed, since it is optimized out.